synapse/lib/stormlib/backup.py

```diff
@@ -0,0 +1,48 @@
+import synapse.common as s_common
+import synapse.lib.stormtypes as s_stormtypes
+
+@s_stormtypes.registry.registerLib
+class BackupLib(s_stormtypes.Lib):
+
+    _storm_lib_path = ('backup',)
+
+    def getObjLocals(self):
+        return {
+            'run': self._runBackup,
+            'list': self._listBackups,
+            'del': self._delBackup,
+        }
+
+    async def _runBackup(self, name=None, wait=True):
+        '''
+        Run a cortex backup.
```

Comment:
Might be worth adding this capability into the devops docs

Reply:
Thus far, we haven't really covered anything in devops docs that are API driven rather than tool driven.  However, now that generating a backup is a cell level capability, we could add it to cmdr, but i'd support that being in a subsequent PR